### PR TITLE
Update database

### PR DESCRIPTION
**Pull request by roza8765**

### CODE REVIEW

**Review by aaitken80 — DISMISSED** [review]
Schema definition matches what was discussed.

**Review by aaitken80 — CHANGES_REQUESTED** [review]
in the TOPIC table document_name and document_link are set to NOT NULL when they are optional fields. I don't know the requirement's here but might need to know the volunteer's name as well as their email address? That was in the previous model.

**Comment by roza8765** [comment]
yes but I think we don't need the volunteer name as we just sent the block to their email

**Comment by roza8765** [comment]
and for the topic part, I think it is required to have  document_name and document_link  as not null

File: server/db/populate-db.sql [hunk 1]
```diff
@@ -4,38 +4,42 @@ INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
 INSERT INTO users (email, password) values ('user@cyf.org', 'user_password');
 
 
- 
- INSERT INTO topics ( topic_name) values ('english');
- INSERT INTO topics ( topic_name) values ('get ready for work');
+INSERT INTO blocks (block_id, block_name) VALUES(1, 'Block 1');
```
Comment (aaitken80) [inline server/db/populate-db.sql]: block_id is set to GENERATED so you cannot insert a value into it here

File: server/db/populate-db.sql [hunk 2]
```diff
@@ -4,38 +4,42 @@ INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
...
+INSERT INTO blocks (block_id, block_name) VALUES(2, 'Block 2');
+
+
+INSERT INTO topics (topic_id, topic_name, document_name, document_link) VALUES(1, 'Beginners English', 'Beginners English', 'www.google.com');
```
Comment (aaitken80) [inline server/db/populate-db.sql]: topic_id is set to GENERATED so you cannot insert a value into it here

File: server/db/populate-db.sql [hunk 3]
```diff
@@ -4,38 +4,42 @@ INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
...
+
+
+
+INSERT INTO block_topics (block_id, topic_id) VALUES(1, 1);
+INSERT INTO block_topics (block_id, topic_id) VALUES(1, 2);
+INSERT INTO block_topics (block_id, topic_id) VALUES(2, 1);
+INSERT INTO block_topics (block_id, topic_id) VALUES(2, 2);
+
+
+INSERT INTO volunteer_topic (volunteer_topic_id, topic_id, volunteer_id) VALUES(1, 1, 1);
```
Comment (aaitken80) [inline server/db/populate-db.sql]: volunteer_topic_id is set to GENERATED so you cannot insert a value into it here.

You are trying to insert a volunteer_id before the volunteer table is populated, you need to move the section 33-34 to above this section.

File: server/db/populate-db.sql [hunk 4]
```diff
@@ -4,38 +4,42 @@ INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
...
+INSERT INTO volunteers (volunteer_id, email) VALUES(1, 'Volunteer1@email.com');
+INSERT INTO volunteers (volunteer_id, email) VALUES(2, 'John@mail.com');
 
-INSERT INTO volunteers ( volunteer_name, email) values ('Shahd','Shahd@cyf.org');
-INSERT INTO volunteers ( volunteer_name, email) values ('Mohammad','mohamad@cyf.org');
-INSERT INTO volunteers ( volunteer_name, email) values ('Razan','razan@cyf.org');
 
-INSERT INTO volunteer_assessments( volunteer_id ,assessment_id, assessment_start, assessment_end) values (2,2,'2020-11-22 11:00:00','2020-11-22 12:15:00');
-INSERT INTO volunteer_assessments( volunteer_id, assessment_id, assessment_start, assessment_end) values (1,1,'2020-11-23 14:30:00','2020-11-23 15:45:00');
+INSERT INTO volunteer_answers (volunteer_topic_id, question_id, answers) VALUES(1, 1, 'Past simple, past continus');
```
Comment (aaitken80) [inline server/db/populate-db.sql]: The column name is answer, not answers here.

**Review by aaitken80 — CHANGES_REQUESTED** [review]

**Review by aaitken80 — APPROVED** [review]

**Pull request merged**